lib/plugins/markdown.py

```diff
@@ -0,0 +1,39 @@
+import os
+import time
+
+
+class markdown_client:
+    def __init__(self, **kwargs):
+        self.save_path = (
+            os.path.join(
+                os.path.dirname(os.path.abspath(__file__)),
```

Comment:
why this? If the user specifies a relative path in the plugins.yml, then that should be relative to the current working dir, and if it's an abspath, don't change it.